src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.ApplyService.Domain.Apply
+{
+    public class RoaptFinancialApplicationsStatusCounts
```

Comment:
Should be RoatpFinancialApplicationsStatusCounts

Reply:
corrected